CHANGES.md

```diff
@@ -42,6 +42,7 @@ Features
 * [#422](https://github.com/twall/jna/pull/422): Update libffi to v3.2.1 - [@tdaitx](https://github.com/tdaitx).
 * [#425](https://github.com/twall/jna/pull/425): Add support for PPCLE - [@tdaitx](https://github.com/tdaix).
 * [#428](https://github.com/twall/jna/pull/428): Added Wincon.h related functions and definitions to `com.sun.jna.platform.win32.Kernel32` - [@lgoldstein](https://github.com/lgoldstein).
+* [#431](https://github.com/twall/jna/pull/431): Added named pipe API support [@lgoldstein](https://github.com/lgoldstein)
```

Comment:
Can you fix it along these lines or do you want me to post a new (squashed) pull request ?